src/Integration/MefServices/ActiveSolutionTracker.cs

```diff
@@ -48,6 +48,21 @@ public ActiveSolutionTracker([Import(typeof(SVsServiceProvider))] IServiceProvid
             ErrorHandler.ThrowOnFailure(this.solution.AdviseSolutionEvents(this, out this.cookie));
         }
 
+        public bool IsSolutionFullyOpened
+        {
+            get
+            {
+                object isLoaded;
+                int hresult = this.solution.GetProperty((int)__VSPROPID4.VSPROPID_IsSolutionFullyLoaded, out isLoaded);
+
+                if (ErrorHandler.Succeeded(hresult) && isLoaded is Boolean)
+                {
+                    return (bool)isLoaded;
+                }
+                return false;
+            }
+        }
```

Comment:
In the previous PR, the class implemented the _IVSSolutionLoadEvents_ interface to get a callback when the solution was fully loaded so it knew when to set the _IsSolutionFullyLoaded_ property.
Since we're not implementing that interface any more this property doesn't need to be in this "tracker" class at all. It could be on one of the other helper classes e.g. _IProjectSystemHelper_.
WDYT?